streams/shared/src/main/scala/zio/stream/ZSink.scala

```diff
@@ -1678,6 +1678,80 @@ object ZSink extends ZSinkPlatformSpecific {
   final val splitLinesChunk: ZSink[Any, Nothing, Chunk[String], Chunk[String], Chunk[String]] =
```

Comment:
Do you mind adding a quick test for this combinator?

Reply:
I did, but I must have misunderstood because it is failing in some cases 🤔

Reply:
Solved